src/Nowruz/pages/setting/sections/account/account.module.scss

```diff
@@ -0,0 +1,50 @@
+@import 'src/components/_primitives.scss';
+
+.cancelBtn {
```

Comment:
No need to write all these styles, please use Button component from Nowruz General Components. In any case you need a special design you can just pass the CustomeStyle props to <Button>